tests/sass/space-around-operator.sass

```diff
@@ -333,6 +333,8 @@ $qux: (2  +1)
 
 $foo: scale-color($foo, $lightness: -14%)
 
+$foobar: #{$foo}-#{$bar}
```

Comment:
Thinking about it, for a partial ident, the previous node only needs to be an interp. It doesn't matter about the next node.